index.js

```diff
@@ -95,13 +96,29 @@ testRunner([
 			Object.keys(refs).forEach(function (uri) {
 				validator.addSchema(uri, refs[uri]);
 			});
+
 			return validator;
 		},
 		test: function (instance, json, schema) {
 			return instance.validate(schema, json) === null;
 		}
 	},
 	{
+		name: 'djv',
+		setup: function (schema) {
+			var validator = djv();
+			var schemas = Object.keys(refs);
+			schemas.splice(3, 1);
```

Comment:
Thanks a lot for your contrbution
This code you have here for setup is against the guidelines for adding validators. The setup should just be using the validator as it comes out of the box.